healthyregions/oeps
# feat: upgrade to NextJS 13.5

## Problem
We would like to upgrade NextJS from v12 to at least v13.5 (but not to v14 just yet)

Fixes #45 

## Approach
* feat: upgrade NextJS to `13.5.6`
    * Migration notes can be found [here](https://nextjs.org/docs/pages/building-your-application/upgrading/version-13)

NOTE: We don't appear to use `<Image>`, `<Link>`, or  `<Script>` anywhere, so much of the migration notes did not seem to apply to our code 🤷‍♀️ 

## How to Test
1. Checkout and run this branch locally
2. Run `yarn install` to update your installed dependencies
3. Run `yarn build` to produce a production build
4. Run `yarn start` to serve the new build on your dev server
5. Navigate to http://localhost:3000 to view the OEPS app
6. Test out the app to make sure there are no residual problems
